src/app/creation/creation.component.html

```diff
@@ -0,0 +1,115 @@
+<form>
+<mat-tab-group mat-align-tabs="start">
+    <mat-tab label="Race">
+        <mat-form-field appearance="fill">
+            <mat-label>Character Race</mat-label>
+            <mat-select [(ngModel)]="selectedRace" name="race" (selectionChange)="displayRaceDescription($event.value)">
+                <mat-option *ngFor="let race of raceData" [value]="race.name">
+                {{race.name}}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+        <div class="body">{{raceDescription}}</div>
+
+    </mat-tab>
+    <mat-tab label="Class">
+        <mat-form-field appearance="fill">
+            <mat-label>Character Class</mat-label>
+            <mat-select [(ngModel)]="selectedClass" name="class" (selectionChange)="displayClassDescription($event.value)">
+                <mat-option *ngFor="let class of classData" [value]="class.name">
+                {{class.name}}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+        <div class="body">{{classDescription}}</div>
+    </mat-tab>
+    <mat-tab label="Attributes">
+        <div class=body>
+            <p>Normally your ability scores are generated randomly, but for new players we recommend you use the Standard array. 
+                Use the following scores and assign them to your attributes: 15, 14, 13, 12, 10, 8.
+            </p>
+        </div>
+        <div class="body" cdkDropListGroup>
+            <div class="example-container">
+              <h2>Strength</h2>
+          
+              <div
```

Comment:
There is a lot of repeated code below this, can we have a list of attributes and generate all of these blocks off of that? Or define an ng-template and just pass a name and id into that?

Reply:
Yes I could probably turn that into a template and just call it. I'll add it as an item in my project so I'll remember.